content/docs/ops/setup/app-health-check/index.md

```diff
@@ -99,55 +79,23 @@ NAME                             READY     STATUS    RESTARTS   AGE
 liveness-6857c8775f-zdv9r        2/2       Running   0           4m
 {{< /text >}}
 
-### Cleanup
-
-Remove the mutual TLS policy and corresponding destination rule added in the steps above:
-
-1. To remove the mutual TLS policy, run:
-
-    {{< text bash >}}
-    $ kubectl delete policies default
-    {{< /text >}}
-
-1. To remove the corresponding destination rule, run:
-
-    {{< text bash >}}
-    $ kubectl delete destinationrules default
-    {{< /text >}}
-
 ## Liveness and readiness probes with HTTP request option
 
-This section shows how to configure health checking with the HTTP request option.
-
-### Mutual TLS is disabled
-
-Run this command to deploy [liveness-http]({{< github_file >}}/samples/health-check/liveness-http.yaml) in the default namespace:
-
-{{< text bash >}}
-$ kubectl apply -f <(istioctl kube-inject -f @samples/health-check/liveness-http.yaml@)
-{{< /text >}}
-
-Wait for a minute and check the pod status to make sure the liveness probes work with '0' in the 'RESTARTS' column.
-
-{{< text bash >}}
-$ kubectl get pod
-NAME                             READY     STATUS    RESTARTS   AGE
-liveness-http-975595bb6-5b2z7c   2/2       Running   0           1m
-{{< /text >}}
+This section shows how to configure health checking with the HTTP request option when mutual TLS is enabled.
```

Comment:
Why is this here if the readers need to do these steps first? Please ensure the order of the instructions works well after removing the content.

Reply:
You mean the steps to enable Istio installation with helm flag `sidecarInjectorWebhook.rewriteAppHTTPProbe=true`? Some apporoach like prober rewrite by annotation does not require this helm flag, same for the separate port.

What do you think? Maybe just mention this flag as an option in `Before you Begin`?